Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -230,11 +230,91 @@ bool Settings::GetHideCursor() const
   return SConfig::GetInstance().bHideCursor;
 }
 
+QString Settings::GetCurrentAudioBackend() const
```

Comment:
What do you mean? It's in Settings because ``Settings`` is supposed to be a wrapper around ``SConfig``.